src/panels/lovelace/views/hui-view.ts

```diff
@@ -20,31 +20,28 @@ import { HomeAssistant } from "../../../types";
 import { classMap } from "lit-html/directives/class-map";
 import { Lovelace, LovelaceCard, LovelaceBadge } from "../types";
 import { createCardElement } from "../create-element/create-card-element";
-import { computeCardSize } from "../common/compute-card-size";
 import { showEditCardDialog } from "../editor/card-editor/show-edit-card-dialog";
 import { HuiErrorCard } from "../cards/hui-error-card";
 import { computeRTL } from "../../../common/util/compute_rtl";
+import { debounce } from "../../../common/util/debounce";
 import { createBadgeElement } from "../create-element/create-badge-element";
 import { processConfigEntities } from "../common/process-config-entities";
 
-let editCodeLoaded = false;
-
-// Find column with < 5 entities, else column with lowest count
-const getColumnIndex = (columnEntityCount: number[], size: number) => {
-  let minIndex = 0;
-  for (let i = 0; i < columnEntityCount.length; i++) {
-    if (columnEntityCount[i] < 5) {
-      minIndex = i;
-      break;
-    }
-    if (columnEntityCount[i] < columnEntityCount[minIndex]) {
-      minIndex = i;
-    }
-  }
-
-  columnEntityCount[minIndex] += size;
-
-  return minIndex;
+let muuri: any;
```

Comment:
TODO: Import types from d.ts files. I think I may need to update TSConfig to look at types in a local folder?

Reply:
I have imported the Types from the Link in the PR above. I have added them to a relative @types path. Not sure this is where we want this? 

No need to update the TSConfig File. Its so smart :) 

How can I type the Muuri variable in the view file without importing Muuri

Reply:
import type

Reply:
MAybe I can use the new import Type that was introduced?